### PR TITLE
Ed25519Signature2020

### PR DESCRIPTION
**Pull request by mattcollier**
See changelog.

Tested in a top-level app in addition to the test suite here.

Waiting on additional changes in cryptold. https://github.com/digitalbazaar/crypto-ld/issues/57

### CODE REVIEW

**Review by dlongley — COMMENTED** [review]
+1 ... this looks like what we discussed, will review again once the crypto-ld changes are available.

File: main.js [hunk 1]
```diff
@@ -97,10 +101,9 @@ export async function verifyCapabilityInvocation({
   // get parsed parameters from from HTTP header and generate signing string
   const {keyId, signingString, params: {signature: b64Signature}} = parsed;
 
-  // fetch verification method from `keyId` and import as a crypto-ld key
-  const verificationMethod = await _getVerificationMethod(
-    {keyId, documentLoader});
-  const key = await LDKeyPair.from(verificationMethod);
+  // fromKeyId ensures that the key is valid and is not revoked
+  const key = await cryptoLd.fromKeyId({id: keyId, documentLoader});
+  const verificationMethod = await key.export({publicKey: true});
```
Comment (dlongley) [inline main.js]:
```suggestion
  const verificationMethod = await key.export({publicKey: true, includeContext: true});
```

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: main.js (hunk 1 above)
Comment (dlongley) [reply main.js]: There's a TODO below "TODO: add parameters to check any other caveats in the capability as..." that isn't a thing anymore. Could you also remove that in this PR?

**Review by dlongley — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (mattcollier) [reply main.js]: @dlongley why do we need the context here?  He consumers of this information have no requirement for @context to be present (evidently).

Separately, I'll remove the TODO.

Comment (dlongley) [reply main.js]: It previously included `@context`, so I'd rather not change it. It gets passed to zcap and jsigs internals which may be expecting `@context` or need to run validation on it, if not now, potentially in the future.

**Pull request merged**